lesson002/task2/task2-2.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html leng="ru">
+    <head>
+        <meta charset="utf-8" />
+        <title></title>
+        <link href="styles/3style.css" rel="stylesheet" >
+    </head>
+    <body>
+        <div id="header1">header1</div>
+        <div id="header2">header2</div>
+        <div id="header3">header3</div>
+        <div id="col1">col1</div>
+        <div id="col2">col2</div>
+        <div id="col3">col3</div>
```

Comment:
Should be full screen. Also in percentage terms.

Reply:
Done